news/2023/2023-02-09-Skin-of-the-Year-2022.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Skin of the Year 2022: Voting Open"
+date: 2023-02-09 09:00:00 +0000
+---
+
+Do you think you got what it takes to select the BEST skin of 2022? Let us know by casting your vote!
+
+![](/wiki/shared/news/2023-02-01-Skin-of-the-Year-2022/SOTY-2022-banner.jpg)
+
+Four score and seven years ago (that being 2007), skinning was first introduced to the then newly released EBA simulator called osu!. Ever since, a sprawling and ever so creative community has been constantly expanding and evolving, many thousands of skins have been uploaded during the past 15 years, many styles and iterations of skins have come and went, but the community never had a "Skin of the Year" competition. Instead of taking a moment to ponder at why that is, let's instead focus on the actual voting in this first-of-a-kind skin competition!
```

Comment:
the joke would've been funny if it was technically accurate tbh

Reply:
are you suggesting this bit gets rewritten then? 2007 is _almost_ 87 years ago if you squint a little bit

Reply:
"1.1568 score and 7 years ago"

```
1.1568 * score + 7 = (today - 2007-09-16 in years)
score ≈ 7.27
```

hmm idk not very elegant

Reply:
i am contractually obligated to mention that i still nose-exhaled while reading it so i think it might still be funny

Reply:
your pick, 0x

Reply:
removed it as per ppy's feedback [1](https://discord.com/channels/188630481301012481/218677502141399041/1072788541978972160) [2](https://github.com/ppy/osu-wiki/pull/8773#discussion_r1099771300)